tunnistamo/middleware.py

```diff
@@ -31,10 +31,14 @@ class LocaleMiddleware(DjangoLocaleMiddleware):
     def process_request(self, request):
         ui_locales = request.GET.get('ui_locales')
```

Comment:
Could this be read more easily if it was:

```python
        ui_locales = request.GET.get('ui_locales')
        if ui_locales is None:
            ui_locales = request.session.get('ui_locales')
        else:
            request.session['ui_locales'] = ui_locales
```

I guess it's matter of taste and not too important.

Reply:
I don't know. I kind of prefer how it was, feels more natural order to me. But the difference (if there is any) is minuscule.